scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -1346,32 +1496,30 @@ fi
 #
 # 2.4.12: XB fixed bugs like keyring is empty + move-back stage now uses params from
 #         my.cnf.
-
-XB_REQUIRED_VERSION="2.4.12"
-
-XB_VERSION=`$INNOBACKUPEX_BIN --version 2>&1 | grep -oe '[0-9]\.[0-9][\.0-9]*' | head -n1`
-if [[ -z $XB_VERSION ]]; then
+#
+XB_2x_VERSION=$($XTRABACKUP_24_PATH/bin/$XTRABACKUP_BIN --version 2>&1 | grep -oe ' [0-9]\.[0-9]\.[0-9]*' | head -n1)
+XB_2x_VERSION=${XB_2x_VERSION# }
+XB_2x_REQUIRED_VERSION="2.4.13"
```

Comment:
Why is 2.4.13 ?

Reply:
Just the latest version released.

Reply:
We tagged to 2.4.12 since it has some important fixes that we banked on. like keyring and all. There is no such significance of 2.4.13 but since this is the first release for 8.0 we can continue to tag it to 2.4.13 being the latest one and we plan to ship release as part of PXC.